src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs

```diff
@@ -72,6 +72,12 @@ public async Task<DisplayVacancyViewModel> GetFullVacancyDisplayViewModelAsync(V
                     await _vacancyDisplayMapper.MapFromVacancyAsync(submittedViewModel, vacancy);
                     submittedViewModel.SubmittedDate = vacancy.SubmittedDate.Value.AsGdsDate();
                     return submittedViewModel;
+                case VacancyStatus.Review:
+                    var reviewViewModel = new ReviewVacancyViewModel();
+                    reviewViewModel.EducationLevelName =
+                        EducationLevelNumberHelper.GetEducationLevelNameOrDefault(programme.EducationLevelNumber, programme.ApprenticeshipLevel);
```

Comment:
```suggestion
                    var reviewViewModel = new ReviewVacancyViewModel{
                    EducationLevelName =
                        EducationLevelNumberHelper.GetEducationLevelNameOrDefault(programme.EducationLevelNumber, programme.ApprenticeshipLevel)
                        };
```

Reply:
very nice to see about the suggestion box.. done